privacy-config/privacy-config-api/src/main/java/com/duckduckgo/privacy/config/api/PrivacyFeatureName.kt

```diff
@@ -27,4 +27,4 @@ enum class PrivacyFeatureName(val value: String) {
     TrackingParametersFeatureName("trackingParameters"),
 }
 
-const val PRIVACY_REMOTE_CONFIG_URL = "https://staticcdn.duckduckgo.com/trackerblocking/config/v4/android-config.json"
+const val PRIVACY_REMOTE_CONFIG_URL = "https://www.jsonblob.com/api/1384585817120497664"
```

Comment:
We probably don't want this merged :)

Reply:
Right, that's why I tagged the commit with `[DO NOT MERGE]` :sweat_smile: just waiting for the test to pass https://github.com/duckduckgo/Android/pull/6254#issuecomment-2984717491.